docs/cluster-management/security.md

```diff
@@ -2,6 +2,12 @@
 
 _WARNING: you are on the master branch, please refer to the docs on the branch that matches your `cortex version`_
 
+## Private cluster
+
+You can configure all instances in your cluster to use private subnets by setting `subnet_visibility: private` in your [cluster configuration](config.md) file before creating your cluster (e.g. via `cortex cluster up -c cluster.yaml`).
+
+You can also configure your operator load balancer (which is what the `cortex` CLI connects to) and/or your API load balancer to be internal by setting `api_load_balancer_scheme: internal` and `operator_load_balancer_scheme: internal` in your cluster configuration file (before creating your cluster). If you do this, you will need to configure [VPC Peering](../guides/vpc-peering.md) to connect to Cortex's VPC.
```

Comment:
Split `operator_load_balancer_scheme` and `api_load_balancer_scheme` to different lines.